packages/terra-data-grid/tests/jest/DataGrid.test.jsx

```diff
@@ -69,21 +67,21 @@ afterAll(() => {
 
 describe('DataGrid', () => {
   it('verifies that the grid created is consistent with the rows and overflowColumns props', () => {
-    const wrapper = shallowWithIntl(
+    const wrapper = mountWithIntl(
       <DataGrid
         id="test-terra-data-grid"
         pinnedColumns={dataFile.cols.slice(0, 2)}
         overflowColumns={dataFile.cols.slice(2)}
         rows={dataFile.rows}
       />,
-    ).dive().dive();
+    );
 
     // One row used for the header.
-    const columnHeader = wrapper.find(ColumnHeader);
+    const columnHeader = wrapper.find('Memo(ColumnHeader)');
```

Comment:
This will look out of place, but shallow doesn't expose the deeply nested component and memoized components don't have display name to be found if you do not have the actual component to compare it with.

Reply:
I don't think this is correct.